### PR TITLE
Include Swiss Governmant Git Repository

### PR DESCRIPTION
**Pull request by Quietghost**
Please also include the official Swiss Governmant Github Repository as well in the landscape "https://github.com/admin-ch/PT-System-Documents" (or also https://github.com/admin-ch/CovidCode-UI and https://github.com/admin-ch/CovidCode-Service) so that not only the Mobile App repository is mentioned - which is only one part of the whole proximity tracing system in Switzerland.

Thank you.

### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] Is your app/project/product currently shipping and/or have the source code available?
* [x] If an app, does your homepage include links to download the Android and/or iOS apps?
* [x] Are you only including a `repo_url` if your project is 100% open source? If so, you need to pick the single best GitHub repository for your project, not a GitHub organization.
* [x] Have you picked the single best (existing) category for your project?
* [x] Does it follow the other guidelines from the [new entries](https://github.com/cncf/landscape#new-entries) section?
* [x] Have you included a URL for your SVG or added it to `hosted_logos` and referenced it there?
* [x] Does your logo clearly state the name of the project/product and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [x] Does your project/product name match the text on the logo?
* [x] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?
* [x] ~10 minutes after opening the pull request, the CNCF-Bot will post the URL for your staging server. Have you confirmed that it looks good to you and then added a comment to the PR saying "LGTM"?

### CODE REVIEW

**Comment by dankohn** [comment]
Thanks for the PR. Could you please list for me every repo that should be included? Is it all of them in https://github.com/admin-ch/ ?

Do we still need to have all of the repos in https://github.com/DP-3T?

**Comment by Quietghost** [comment]
Basically the PTS (Proximity Tracing System) of Switzerland consists of two parts:

1. The DP-3T part (internally we speak of the red system) which includes the mobile app (iOS and Android) as well as the backend and a config service. That you can find in the mentioned repo  https://github.com/DP-3T

2. The Swiss government provides the system (internally black system) for the medical stuff to generate codes that the people who want to report that they have the Covid-19 virus in the mobile app can use to do so - otherwise it doesn`t work. This part you can fin in the https://github.com/admin-ch/CovidCode-UI and https://github.com/admin-ch/CovidCode-Service repos.

3. A documentation of the Swiss implementation of PTS can be found in https://github.com/admin-ch/PT-System-Documents

So basically we need both repos (DP3-T and Admin-CH) to tell the complete picture.

**Comment by dankohn** [comment]
Please confirm that I got the list correct: https://landscape.lfph.io/selected=switzerland-swiss-covid

If I have anything wrong, you can let me know here or please feel free to open a new PR. You can see the syntax of one repo per line.
